src/components/Header.js

```diff
@@ -40,8 +44,23 @@ export default function(store){
         if (/^\d{5}$/.test(newZipcode) || !newZipcode) {
           store.dispatch('setFilters',{zipcode: newZipcode });
         }
+      },
+      headerHeight: function() {
+        return (this.$refs.header) ? this.$refs.header.clientHeight : null;
+      },
+      checkIfHeaderShouldBeStuck() {
```

Comment:
Naming nit / suggestion, since this is not just checking but also setting the header's stickiness or non-stickiness, maybe something like `setStickyHeader` ?